app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,67 @@
+class InvitesController < ApplicationController
+  before_action :no_loggedin
```

Comment:
hm?

Reply:
can't be invited if you're already invited :P

Reply:
okaybut I believe the name of the method could be improved a smidge

Reply:
_okayfair_